software/verification/verify_cdl_sequence.py

```diff
@@ -94,23 +96,53 @@ def parse_configurations(self):
                 else:
                     reference['modelJsonDirectory'] = default_model_json_directory
 
-            if not reference['modelJsonDirectory'].endswith('/'):
-                reference['modelJsonDirectory'] += '/'
+            if not reference['modelJsonDirectory'].endswith(os.sep):
+                reference['modelJsonDirectory'] += os.sep
 
             if ref_run_controller_flag:
                 reference['controller'] = self.real_controller
 
             self.test_list.append(reference)
 
+    def run_modelica_json(self, model, model_dir):
+        """this function uses the modelica-json tool to convert modelica files to json
```

Comment:
Better start comments in capital. Otherwise, you can't use proper punctuation if the comment is more than one sentence. (The python library also uses full sentences or paragraphs.)